---
title: "Resume"
author:
  - name: Wen-Ching Calvin Chan, Ph.D.
    email: wenching.chan@gmail.com
package: "Calvin's Resume"
bibliography: Resume.bib
output:
  # pdf_document:
  #   highlight: tango
  #   number_sections: true
  #   df_print: kable # default, kable, tibble
  #   toc: true
  #   toc_depth: 3
  #   latex_engine: xelatex
  #   keep_tex: yes
  #   includes:
  #     in_header: header.tex
  
  html_document:
    theme: united
    highlight: tango
    number_sections: true
    df_print: paged
    toc: true
    toc_depth: 3
    toc_float:
      collapsed: true
      smooth_scroll: false
    keep_md: true
  
  # BiocStyle::pdf_document:
  #   df_print: paged
  
  # BiocStyle::html_document:
  #   df_print: paged
always_allow_html: yes
vignette: >
  %\VignetteIndexEntry{JobInterview}
  %\VignetteEngine{knitr::rmarkdown}
  %\VignetteEncoding{UTF-8}
---



<a name="Top"/>


# CALVIN'S RESUME

***

This is a public repository for my simple resume at [wenching.github.io](https://wenching.github.io/) and more details about me below.


# WHO AM I

***

> An enthusiastic and highly motivated research scientist with 10+ years of experience of focusing on Bioinformatics.
> Strong research professional with extensive skills of integrating multi '-omics' and clinical data using statistical, data mining, and machine learning techniques.
> An inspiring, productive employee possesses a keen ability for implementing and learning new emerging Bioinformatics technologies.



# RESEARCH HIGHLIGHTS

***

## FY19

- CRI-BIO-448/480
    - Project title
        - Meis Proteins identify genes associated with Prostate Cancer Progression (Meis in Prostate Cancer Progression)
    - PI/Client
        - [Donald Vander Griend](https://pathology.uic.edu/directory/donald-j-vander-griend-phd/), Assistant Professor, Director of Urological Research, <dvanderg@surgery.bsd.uchicago.edu>
    - Department
        - Surgery
    - Short project description/Bioinformatics analysis goals/Assistance provided:
        - Given that MEIS proteins have been found to be prognostic in Prostate cancer (PCs), this study focuses on exploring the functional and predictive role of changes in MEIS expression associated with prostate tumor progression. A sophisticated analysis of RNA-seq dataset of primary prostate tumor samples was performed. All samples were first stratified into two groups high and low based on MEIS1/MEIS2 expressions. A meta-comparison of identifying MEIS associated genes involved in cancer progression was followed based on differentially expressed genes between these two groups. Coupling with further analyses based on RNA-seq results, our results demonstrate a stepwise decrease in the expression of both MEIS1 and MEIS2 from benign epithelia, to the primary tumor, to metastatic tissues. The work was published in a high-impact journal **_Clinical Cancer Research_** (2017 IF: 10.199) in 2018.
    - Results: 
        <!-- * Manuscript published/accepted/submitted; Grant application prepared/submitted. -->
        - The paper was published as follows;
        - Bhanvadia, R.R., VanOpstall, C., Brechka, H., Barashi, N.S., Gillard, M., McAuley, E.M., Apiz, J., Paner, G.P., **`Chan, W.-C.`**, Andrade, J., et al. (2018). MEIS1 and MEIS2 Expression and Prostate Cancer Progression: A Role For HOXB13 Binding Partners in Metastatic Disease. Clin. Cancer Res. (2017 IF: 10.199)
- CRI-BIO-576
    - Project title
        - Uncovering the genetic and functional identity of leukemia stem cells (Zebrafish T-ALL LSCs)
    - PI/Client
        - [Jill de Jong](https://www.uchicagomedicine.org/find-a-physician/physician/jill-de-jong), Assistant Professor, <jdejong@peds.bsd.uchicago.edu>
    - Department
        - Pediatrics
    - Short project description/Bioinformatics analysis goals/Assistance provided:
        - In this study, the "side population assay" (enriches for Leukemia Stem Cells (LSCs)) was utilized to sort cells from zebrafish T-ALLs and run a high-throughput transcriptome profiling application (i.e., RNA sequencing) on the "side population" and the "main population". The null hypothesis was to identify LSC associated genes and pathways that may be driving LSC activity. In order to achieve this goal, a computational comparison between the gene expression profiles of the two groups was performed. A comprehensive, computational analysis of RNA-seq dataset was conducted, including quality assessment, tag alignment, transcriptional quantification, differential expression analysis, functional enrichment analysis, and data visualization. Overall, our results demonstrate that the side population in zebrafish acute lymphoblastic leukemia significantly enriches for leukemia propagating cells and identifies the Wnt-pathway as a likely genetic driver of leukemia stem cell fate. The work was published in a high-impact journal **_Haematologica_** (2017 IF: 9.09) in 2019.
    - Results: 
        <!-- * Manuscript published/accepted/submitted; Grant application prepared/submitted. -->
        - The paper was published as follows;
        - Baeten, J.T., Waarts, M.R., Pruitt, M.M., **`Chan, W.-C.`**, Andrade, J., and Jong, J.L.O. de (2019). The side population enriches for leukemia propagating cell activity and Wnt pathway expression in zebrafish acute lymphoblastic leukemia. Haematologica haematol.2018.206417.




# ROAD MAP  

***

- 2015.10 ~ present
    - Bioinformatician II, Center for Research Informatics, University of Chicago, USA
    - Publications
        - Domowicz, M., **`Chan, W.-C.`**, Claudio-Vázquez, P., Henry, J., Ware, C., Andrade, J., Dawson, G., and Schwartz, N. Global brain transcriptome analysis of a Tpp1 Neuronal Ceroid Lipofuscinoses Mouse Model. ASN Neuro. 2019 doi: 10.1177/1759091419843393. (2017 IF: 3.617) [@domowicz_global_2019]
        - Baeten, J.T., Waarts, M.R., Pruitt, M.M., **`Chan, W.-C.`**, Andrade, J., and Jong, J.L.O. de (2019). The side population enriches for leukemia propagating cell activity and Wnt pathway expression in zebrafish acute lymphoblastic leukemia. Haematologica haematol.2018.206417. (2017 IF: 9.09) [@baeten_side_2019]
        - Kline, D.E., MacNabb, B.W., Chen, X., **`Chan, W.-C.`**, Fosco, D., and Kline, J. (2018). CD8$\alpha$+ Dendritic Cells Dictate Leukemia-Specific CD8+ T Cell Fates. The Journal of Immunology ji1801184. (2017 IF: 4.539) [@kline_cd8+_2018]
        - Bhanvadia, R.R., VanOpstall, C., Brechka, H., Barashi, N.S., Gillard, M., McAuley, E.M., Apiz, J., Paner, G.P., **`Chan, W.-C.`**, Andrade, J., et al. (2018). MEIS1 and MEIS2 Expression and Prostate Cancer Progression: A Role For HOXB13 Binding Partners in Metastatic Disease. Clin. Cancer Res. (2017 IF: 10.199) [@bhanvadia_meis1_2018]
        - Wang, L.-J., **`Chan, W.-C.`**, Chou, M.-C., Chou, W.-J., Lee, M.-J., Lee, S.-Y., Lin, P.-Y., Yang, Y.-H., and Yen, C.-F. (2017). Polymorphisms of STS gene and SULT2A1 gene and neurosteroid levels in Han Chinese boys with attention-deficit/hyperactivity disorder: an exploratory investigation. Sci Rep 7, 45595. (2017 IF: 4.122) [@wang_polymorphisms_2017]
        - Kach, J., Long, T.M., Selman, P., Tonsing-Carter, E.Y., Bacalao, M.A., Lastra, R.R., Wet, L. de, Comiskey, S., Gillard, M., VanOpstall, C., West D.C., **`Chan, W.-C.`**, et al. (2017). Selective Glucocorticoid Receptor Modulators (SGRMs) Delay Castrate-Resistant Prostate Cancer Growth. Mol Cancer Ther 16, 1680–1692. (2017 IF: 5.365) [@kach_selective_2017]
    - Seminar Series
        - [Genomics	Data	Visualization](ftp://logia.cri.uchicago.edu/bioinformatics/tutorials/May2017/CRI_Training.pdf)
    - Internal Project - RNA-seq Pipeline implementing using Python, R, and BigDataScript
        - [2018 CRI RNA-seq Pipeline](https://github.com/wenching/cri_rnaseq_2018/tree/master/docs)
- 2015.01 ~ 2015.09
    - Senior Postdoctoral Fellow, Department of Medical Research, Kaohsiung Chang Gung Memorial Hospital, Taiwan
    - Publications
        - Li, S.-C., **`Chan, W.-C.`**, Huang, Y.-H., Guo, M.M.-H., Yu, H.-R., Huang, F.-C., Kuo, H.-C., and Kuo, H.-C. (2016). Major methylation alterations on the CpG markers of inflammatory immune associated genes after IVIG treatment in Kawasaki disease. BMC Med Genomics 9. [@li_major_2016]
        - Kuo, H.-C., Li, S.-C., Guo, M.M.-H., Huang, Y.-H., Yu, H.-R., Huang, F.-C., Jiao, F., Kuo, H.-C., Andrade, J., and **`Chan, W.-C.`** (2016). Genome-Wide Association Study Identifies Novel Susceptibility Genes Associated with Coronary Artery Aneurysm Formation in Kawasaki Disease. PLOS ONE 11, e0154943. [@kuo_genome-wide_2016]
        - Kuo, H.-C., Hsieh, K.-S., Ming-Huey Guo, M., Weng, K.-P., Ger, L.-P., **`Chan, W.-C.`**, and Li, S.-C. (2016). Next-generation sequencing identifies micro-RNA–based biomarker panel for Kawasaki disease. Journal of Allergy and Clinical Immunology 138, 1227–1230. [@kuo_next-generation_2016]
    - Awards
        - [The 13th National Innovation Award](https://innoaward.taiwan-healthcare.org/AwardDetail.php?REFDOCTYPID=0nimiycqow7g4bru&NumID=0oixv0ge1k07vc47&REFDOCID=0oj6w0yr9wyk0k5c) & [Certificate](IMG/2016NationalInnovationAwardCertificate.png)
- 2011.10 ~ 2013.08
    - Publications
        - Sofueva, Sevil, Eitan Yaffe, **`Wen-Ching Chan`**, Dimitra Georgopoulou, Matteo Vietri Rudan, Hegias Mira-Bontenbal, Steven M Pollard, Gary P Schroth, Amos Tanay, and Suzana Hadjur. 2013. “Cohesin-Mediated Interactions Organize Chromosomal Domain Architecture.” The EMBO Journal 32 (24): 3119–29. [@sofueva_cohesin-mediated_2013]


[Top](#Top)

***


